src/Raven.Server/Documents/Indexes/IndexStorage.cs

```diff
@@ -1245,6 +1245,40 @@ internal static void WriteVectorDimensions(RavenTransaction tx, Dictionary<strin
             foreach (var kvp in vectorDimensionsToAdd)
                 fieldsTree.Add(kvp.Key, kvp.Value.ToString());
         }
+
+        internal Dictionary<string, string> ReadVectorSourceAiTaskIdentifiers()
```

Comment:
What is the meaning of this?

Reply:
Renamed to `ReadEmbeddingsGenerationTaskIdentifiers`
Generally, we want to ensure that only one task is used for the field, and we also need it during the query.